src/staging/src/hooks/useContent.js

```diff
@@ -0,0 +1,53 @@
+import { useState, useEffect } from "react";
+import * as cheerio from "cheerio";
+
+export const useContent = (contentHtml) => {
+  const [title, setTitle] = useState("");
+  let video_id;
+  let media; 
+  let getIframe
+
+  if(contentHtml?.introHtml){
+    let $ =  cheerio.load(contentHtml?.introHtml);
+    getIframe = $("iframe");
+    media = $(".media");
```

Comment:
If this works like jQuery, then I think this selector could be more specific and use `figure.media` instead of `.media`

Also the media is a child/descendant of the iframe.  Is there a child selector you can use from the iframe, so you don't accidentally get a different `figure.media` from somewhere else on the page?

Reply:
@molund 
thank you for testing, helping to find and giving solutions! 
in any way better to add those cases, I will add  

1) Multiple options toAraay 
2) Specify elem by `figure.media`